docs/README.md

```diff
@@ -24,9 +26,13 @@ poetry install
 poetry run zenml integration install -f
 poetry run pip install click~=8.0.3 typing-extensions~=3.10.0.2
 ```
-* Run  `poetry run bash scripts/generate-docs.sh` from the repository root
+* Run  `poetry run bash scripts/serve_api_docs.sh` from the repository root - 
```

Comment:
I get the following `BuildError` when I run this script. I followed all the other steps including a `poetry update` etc.

```
+ cd docs
+ mkdocs serve
INFO     -  Building documentation...
INFO     -  Cleaning site directory
ERROR    -  mkdocstrings.extension: <module 'zenml.types' (namespace)> is a built-in module
            Traceback (most recent call last):
              File "/Users/strickvl/.pyenv/versions/3.8.6/envs/zenml-dev-3.8.6/lib/python3.8/site-packages/pytkdocs/cli.py", line 205, in main
                output = json.dumps(process_json(line))
              File "/Users/strickvl/.pyenv/versions/3.8.6/envs/zenml-dev-3.8.6/lib/python3.8/site-packages/pytkdocs/cli.py", line 114, in process_json
                return process_config(json.loads(json_input))
              File "/Users/strickvl/.pyenv/versions/3.8.6/envs/zenml-dev-3.8.6/lib/python3.8/site-packages/pytkdocs/cli.py", line 91, in process_config
                obj = loader.get_object_documentation(path, members)
              File "/Users/strickvl/.pyenv/versions/3.8.6/envs/zenml-dev-3.8.6/lib/python3.8/site-packages/pytkdocs/loader.py", line 358, in get_object_documentation
                root_object = self.get_module_documentation(leaf, members)
              File "/Users/strickvl/.pyenv/versions/3.8.6/envs/zenml-dev-3.8.6/lib/python3.8/site-packages/pytkdocs/loader.py", line 397, in get_module_documentation
                source = Source(inspect.getsource(module), 1)
              File "/Users/strickvl/.pyenv/versions/3.8.6/lib/python3.8/inspect.py", line 985, in getsource
                lines, lnum = getsourcelines(object)
              File "/Users/strickvl/.pyenv/versions/3.8.6/lib/python3.8/inspect.py", line 967, in getsourcelines
                lines, lnum = findsource(object)
              File "/Users/strickvl/.pyenv/versions/3.8.6/lib/python3.8/inspect.py", line 780, in findsource
                file = getsourcefile(object)
              File "/Users/strickvl/.pyenv/versions/3.8.6/lib/python3.8/inspect.py", line 696, in getsourcefile
                filename = getfile(object)
              File "/Users/strickvl/.pyenv/versions/3.8.6/lib/python3.8/inspect.py", line 659, in getfile
                raise TypeError('{!r} is a built-in module'.format(object))
            TypeError: <module 'zenml.types' (namespace)> is a built-in module
ERROR    -  Error reading page 'index.md':
ERROR    -  Could not collect 'zenml.types'

Aborted with a BuildError!
```

Reply:
Spoke with @AlexejPenner and we figured out the main thing going wrong. `mkdocs` gets confused by empty folders, but he'll amend the script to take that into account. There was another error which seems localised to my machine which we'll ignore for now.

Reply:
This is due to empty directories in your local environment. These empty directories happen as a byproduct of our refactorings. I fixed this now by ignoring empty directories.

Reply:
Fixed this now :)